server/athenian/api/openapi/openapi.yaml

```diff
@@ -1083,6 +1260,91 @@ components:
       items:
         $ref: '#/components/schemas/DeveloperID'
       type: array
+    TeamMember:
+      description: A team member.
+      example:
+        login: github.com/gkwillie
+        id: github|60340680
+        native_id: 60340680
+        name: Groundskeeper Willie
+        email: bot@athenian.co
+        picture: https://avatars0.githubusercontent.com/u/60340680?v=4
+        updated: 2020-01-23T12:00:00Z
+      allOf:
+        - $ref: '#/components/schemas/User'
+        - properties:
+            login:
```

Comment:
How about adding this to the `User` model? Thus we can delete this `allOf`.

Reply:
Indeed, probably that would the best thing. 👍